docs/develop/make-office-web-add-in-compatible-with-existing-com-add-in.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Make your Office Web Add-in compatible with an existing COM add-in-in
+description: 'Enable compatibility with an equivalent COM add-in that has the same functionality as your Office web add-in'
+ms.date: 04/22/2019
+localization_priority: Normal
+---
+
+# Make your Office Web Add-in compatible with an existing COM add-in
+
+If you have an existing COM add-in, you can build equivalent functionality in your Office web add-in to extend your solution features to other platforms such as online or macOS. However, Office web add-ins don't have all of the functionality available in COM add-ins. Depending on the functionality your solution uses, the COM add-in may provide a better experience than the web add-in on Windows in Excel, Word, and PowerPoint.
```

Comment:
Consider "Depending on the functionality your solution uses" -> "Depending on your solution"
The word functionality gets used a lot, so this cuts back on the repetition.

Reply:
@AlexJerabek actually simplified even further. "Your COM add-in may provide a better experience..." Thanks!